Sources/protoc-gen-swift/ExtensionGenerator.swift

```diff
@@ -115,10 +115,13 @@ struct ExtensionGenerator {
         let scope = swiftDeclaringMessageName == nil ? "" : "static "
         let traitsType = descriptor.getTraitsType(context: context)
 
-        var fieldNamePath = self.protoPath
-        if fieldNamePath.hasPrefix(".") {
-            fieldNamePath.remove(at: fieldNamePath.startIndex)
-            fieldNamePath = "\"\(fieldNamePath)\""
```

Comment:
By convention, the package name inside of protoc always begins with a '.'.  So this path always gets taken.

I should probably just assert the presence of the leading '.' instead of making this conditional.